uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/LeaveRequestTest.php

```diff
@@ -2065,4 +2070,59 @@ public function testLeaveRequestIsDeletedValueCanNotBeSetWhenCreatingALeaveReque
     $leaveRequestRecord = LeaveRequest::findById($leaveRequest->id);
     $this->assertEquals(0, $leaveRequestRecord->is_deleted);
   }
+
+  public function testEmailGetsSentWhenLeaveRequestIsCreated() {
```

Comment:
mmmmm... since both tests are pretty much the same, what about using the same test for both? After checking that the messages were sent on create, we empty the spool table, update the leave request and then check if new messages were sent.

Usually it's better to tests things separately, but given the amount of code duplicate here and that we're actually testing the same lines of code, I think it's fine to make an exception